### PR TITLE
Added 7 new easy questions

### PR DESCRIPTION
**Pull request by lucyturn3r**
Added 7 new easy questions - 1 debugging, the rest are functions or programs

### CODE REVIEW

File: codewof/programming/content/en/square-perimeter/question.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Square Perimeter
+
+Write a function `square_perimeter(side_length)` that **returns** the perimeter of the square where the length of a side
+is `side_length`. The formula for calculating the perimeter of a square is `p = 4s` where `p` is the perimeter and `s` 
+is the length of a side.
```
Comment (eAlasdair) [inline codewof/programming/content/en/square-perimeter/question.md]: again, *length of* permimeter

File: codewof/programming/content/en/best-paper-scissors-rock/question.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Best Paper, Scissors, Rock
+
+Write a program that plays one round of paper, scissors, rock with a user. It should ask the user whether they would 
+like to play paper, scissors, or rock, and then your program should then **print** out what it will play in return.
```
[the remark this answers is not in the document]
Comment (lucyturn3r) [reply codewof/programming/content/en/best-paper-scissors-rock/question.md]: I have gone through and changed all the questions added in this PR to match this

File: codewof/programming/content/en/best-paper-scissors-rock/question.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Best Paper, Scissors, Rock
+
+Write a program that plays one round of paper, scissors, rock with a user. It should ask the user whether they would 
+like to play paper, scissors, or rock, and then your program should then **print** out what it will play in return. 
+Your program should always win, i.e.
```
[the remark this answers is not in the document]
Comment (lucyturn3r) [reply codewof/programming/content/en/best-paper-scissors-rock/question.md]: I have changed the 'i.e.' to ' by following these rules', and added a colon to the end

File: codewof/programming/content/en/square-area/question.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# Square Area
+
+Write a function `square_area(side_length)` that **returns** the area of the square where the length of a side is `side_length`. 
+The formula for calculating the area of a square is a = s<sup>2</sup> where `a` is the area and `s` is the length of a side.
```
Comment (eAlasdair) [inline codewof/programming/content/en/square-area/question.md]: Could you put the formula in backticks to be consistent?

File: codewof/programming/content/en/square-perimeter/question.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Square Perimeter
+
+Write a function `square_perimeter(side_length)` that **returns** the length of the perimeter of the square where the length of a side is `side_length`. 
+The formula for calculating the length of the perimeter of a square is `p = 4s` where `p` is the length of the perimeter and `s`  is the length of a side.
```
Comment (eAlasdair) [inline codewof/programming/content/en/square-perimeter/question.md]: Double space between '`s`' and 'is'; not at all important

File: codewof/programming/content/en/square-area/question.md (hunk 4 above)
Comment (lucyturn3r) [reply codewof/programming/content/en/square-area/question.md]: The only reason I didn't put it in as a code block was that superscripts cannot be used in code blocks - and I didn't want to literally give them the code. The options were:
- `a = s ** 2` which gives away the main point of the excercise
- `a = s ^ 2` which is confusing
- a = s<sup>2</sup> which is the notation they are used to but without the formatting placed on the other formulas
- (or `a = s`<sup>2</sup> but that is just ugly)

Do you think that `a = s ** 2` should be used?

Comment (eAlasdair) [reply codewof/programming/content/en/square-area/question.md]: I see, okay in that case not having backticks is fine

**Pull request merged**